spec/requests/api/v0/vendors_request_spec.rb

```diff
@@ -59,7 +59,7 @@
 
       ### Alternate test to check for destroyed Vendor
 
-      vendor2 = create(:vendor)
+      vendor2 = create(:vendor, credit_accepted: true)
 
```

Comment:
Is there a reason why we are explicitly stating the :credit_accepted value?

Reply:
sorry I did when testing my validations but we do not need this anymore or the commented out validation in the vendor model. I am going to push those changes up now